recipes/xdg/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "xdg" %}
+{% set version = "1.0.5" %}
+{% set sha256 = "d495c8c620a6a34b6111f3254d6fab5f06c1a7ce532e8f34e029209a19e6ce65" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
```

Comment:
Does this package work with windows? In the meta info in the `setup.py`, it only mentions `unix`, and I don't think `expandvars` in the following behaves as expected on windows:

https://github.com/srstevenson/xdg/blob/master/xdg.py#L60

If you need to add `skip: True  # [win]`, then the package should no longer be `noarch: python`.

Reply:
yeah this likely doesn't work on windows. The expandvars behavior should be fine according to the [docs](https://docs.python.org/2.7/library/os.path.html#os.path.expandvars). But I doubt the variables are defined on windows.